src/documents/activities/MatchingGame.md

```diff
@@ -6,17 +6,18 @@ eleventyNavigation:
     order: 1
 ---
 
-Ask someone to think of and briefly describe someone they know (their age, interests, where they live, etc), then ask someone else to describe a daily task or goal that they typically have. Consider the context or situation in which your persona is carrying out their task or goal (e.g. at home in the morning, or in the car on the highway, etc). Feel free to add any information that might help to describe the situation in more detail. Consider an existing product or service that might help your persona accomplish their goal in the given context.
+Ask someone to think of and briefly describe someone they know (their age, interests, where they live, etc), then ask
+someone else to describe a daily task or goal that they typically have. Consider the context or situation in which your
+persona is carrying out their task or goal (e.g. at home in the morning, or in the car on the highway, etc). Feel free
+to add any information that might help to describe the situation in more detail. Consider an existing product or service
+that might help your persona accomplish their goal in the given context.
 
-Frame your persona’s goal into a list of needs by considering the step-by-step process they might take in accomplishing their goal.  Where possible, match these needs up with the features associated with your product or service.
+Frame your persona’s goal into a list of needs by considering the step-by-step process they might take in accomplishing
+their goal.  Where possible, match these needs up with the features associated with your product or service.
 
-Decide whether or not your persona was able to
-accomplish their goal using the product or service
-you have selected. Were their needs met at each
-step? If not, consider how you might redesign this
-product or service in a way that would allow them to
-succeed, keeping in mind the insights, practices
-and tools of inclusive design.
+Decide whether or not your persona was able to accomplish their goal using the product or service you have selected.
+Were their needs met at each step? If not, consider how you might redesign this product or service in a way that would
+allow them to succeed, keeping in mind the insights, practices and tools of inclusive design.
```

Comment:
Something for a future ticket, the list is hard to read on the site because there are no visible bullets, also when they wrap to multiple lines it is hard to distinguish between separate points. @jhung thoughts on this?

Reply:
@jobara That's just a paragraph, isn't it?

Reply:
See the published page: https://guide.inclusivedesign.ca/activities/MatchingGame.html

Reply:
:( I went back and highlighted the wrong section.. got confused by the "-" from the PR.. I'll add my comment to a different spot.